packages/google_mlkit_commons/android/src/main/java/com/google_mlkit_commons/InputImageConverter.java

```diff
@@ -39,16 +40,27 @@ public static InputImage getInputImageFromData(Map<String, Object> imageData,
                     Map<String, Object> metaData = (Map<String, Object>) imageData.get("metadata");
 
                     assert metaData != null;
-                    inputImage = InputImage.fromByteArray((byte[]) Objects.requireNonNull(imageData.get("bytes")),
-                            Double.valueOf(Objects.requireNonNull(metaData.get("width")).toString()).intValue(),
-                            Double.valueOf(Objects.requireNonNull(metaData.get("height")).toString()).intValue(),
-                            Integer.parseInt(Objects.requireNonNull(metaData.get("rotation")).toString()),
-                            Integer.parseInt(Objects.requireNonNull(metaData.get("image_format")).toString()));
-                    return inputImage;
+                    byte[] data = (byte[]) Objects.requireNonNull(imageData.get("bytes"));
+                    int imageFormat = Integer.parseInt(Objects.requireNonNull(metaData.get("image_format")).toString());
+                    int rotationDegrees = Integer.parseInt(Objects.requireNonNull(metaData.get("rotation")).toString());
+                    int width = Double.valueOf(Objects.requireNonNull(metaData.get("width")).toString()).intValue();
```

Comment:
use `Integer.parseInt` for these 2 as well

Reply:
The underlying dart type in InputImageMetadata is Size.height, which uses double unfortunately.